### ISSUE TITLE
Clarify bazelbuild/migration-tooling VS johnynek/bazel-deps

### ISSUE THREAD

**Issue by vorburger** [bazelbuild/migration-tooling#82]
https://docs.bazel.build/versions/master/generate-workspace.html points to this repo.

But the code in this repo appears to be broken, and seems to have been in a while.

There are folks opening issues in this repo and it doesn't look like anyone is managing it?

https://github.com/bazelbuild/migration-tooling/pull/80 (for https://github.com/bazelbuild/migration-tooling/issues/47) says "we've switched to https://github.com/johnynek/bazel-deps instead".

Should this repo be removed and the documentation updated?

Or should more people be made committers in this repo?

**Comment by vorburger** [comment]
@johnynek @nevillelyh FYI

**Comment by johnynek** [comment]
I have offered on the mailing list to give the bazel-deps repo to bazelbuild and I'm happy to add more owners/committers.

**Comment by davidstanke** [comment]
Yes, I expect we will move bazel-deps to bazelbuild (thank you @johnynek!)... we need a few days to work through some process on our end.

**Comment by jspawar** [comment]
I'm not sure I understand why this will be replaced by `bazel-deps`? From my quick scan of that project, it doesn't really provide a means of migrating a Maven project to Bazel. Mapping my `pom.xml` to some YAML file and then using that utility doesn't seem like a preferable migration path

**Comment by johnynek** [comment]
There is a migration tool in that repo as well which converts a pom file to the yaml it expects but it is underdocumented.

**Comment by jspawar** [comment]
Ah ok, I'm guessing you're talking about `//src/scala/com/github/johnynek/bazel_deps/maven:maven_tool`?

**Comment by johnynek** [comment]
Yes.

**Comment by oferb** [comment]
For anyone interested in johnynek/bazel-deps, here's an example usage with some explanations and helpful scripts: https://github.com/google/startup-os/tree/master/tools/deps
Thanks Johny!
